share/man/man9/intro.9

```diff
@@ -450,10 +450,11 @@ Kernel sanitizers can perform additional compiler-assisted checks against
 memory use/access.
 These runtimes are capable of detecting difficult-to-identify classes of bugs,
 at the cost of a large overhead.
-Supported is the Kernel Address Sanitizer
-.Xr KASAN 9 ,
-and the Kernel Memory Sanitizer
-.Xr KMSAN 9 .
+The Kernel Address Sanitizer
+.Xr KASAN 9
+and Kernel Memory Sanitizer
+.Xr KMSAN 9
+are supported.
 .Pp
 The
 .Cd LOCK_PROFILING
```

Comment:
Is `.Cd` markup essential, or can this sentence change? To not have LOCK_PROFILING repeated (as an `.Xr`) at the tail: 

> The LOCK_PROFILING(9) kernel config option enables extra code to assist with profiling and/or debugging lock performance.

Reply:
Sure it can change to the latter.

Reply:
Resolved by [`5ed65c5` (#813)](https://github.com/freebsd/freebsd-src/pull/813/commits/5ed65c5dfb168a5c02be5c574b9ecf78c0864958)